collectors/0/kafka_consumer_jmx.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import re
+import sys
+
+from collectors.lib.jmx_monitor import JmxMonitor
+
+MONITORED_MBEANS = ["kafka.consumer", ""]
+
+
+class KafkaConsumerJmxMonitor(JmxMonitor):
+    USER = "flume"
+    PROCESS_NAME = "org.apache.flume.node.Application"
+    METRIC_PREFIX = "flume"
```

Comment:
Yes, the collectors are running on a couple of Flume consumers and producers, so I've verified that they work. 

I agree with the refactor - only thing I would disagree with is leaving flume_kafka as one role, since I cannot envision any scenario in which we would want to run producers and consumers on the same host.